### DISCUSSION TITLE
App only exits sometimes

### DISCUSSION THREAD

**Discussion by DasLixou** [bevyengine/bevy#5906] [Q&A] [answered]
I just use the DefaultPlugins and when i close the window, sometimes the console also exits but sometimes i have to CTRL+C manually. Why and how can i fix this?

**Comment by nicopap** [comment, marked as answer]
Are you on linux? This is a known issue.
* https://github.com/bevyengine/bevy/issues/5524

**Reply by DasLixou** [reply]
Yep - archlinux, x11, amd radeon. so no xwayland but still error
